src/antworld/route_continuous.cc

```diff
@@ -246,35 +246,35 @@ void RouteContinuous::render(meter_t antX, meter_t antY, degree_t antHeading) co
 
     glBindVertexArray(m_OverlayVAO);
 
-    glTranslatef(static_cast<GLfloat>(antX.value()), static_cast<GLfloat>(antY.value()), 0.1f);
-    glRotatef(static_cast<GLfloat>(-antHeading.value()), 0.0f, 0.0f, 1.0f);
+    glTranslatef(static_cast<GLfloat>(pose.x().value()), static_cast<GLfloat>(pose.y().value()), 0.0f);
+    glRotatef(static_cast<GLfloat>(-pose.yaw().value()), 0.0f, 0.0f, 1.0f);
     glDrawArrays(GL_LINES, 0, 2);
     glPopMatrix();
 
     // Unbind VAOs
     glBindVertexArray(0);
 }
 //----------------------------------------------------------------------------
-bool RouteContinuous::atDestination(meter_t x, meter_t y, meter_t threshold) const
+bool RouteContinuous::atDestination(const Vector2<meter_t> &position, meter_t threshold) const
 {
     // If route's empty, there is no destination so return false
     if(m_Waypoints.empty()) {
         return false;
     }
     // Otherwise return true if
     else {
-        return distance(m_Waypoints.back(), x, y) < threshold;
+        return distance(m_Waypoints.back(), position.x(), position.y()) < threshold;
```

Comment:
You could also do ``m_Waypoints.back().distance2D(position)`` instead.

Reply:
Sadly that doesn't work as ``m_Waypoints`` are passed directly to OpenGL so are unitless ``GLfloat``s